### PR TITLE
docs: update overview page

### PR DESCRIPTION
**Pull request by kv-chiu**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Selflocking plz review

File: docs/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.gradientborder {
+  margin: 0 auto;
+  transform: scale(0.85);
+  width: 600px;
+  box-shadow: 0 0 50px grey;
+  border-radius: 20px;
+}
+
+.signingradientborder {
```
Comment (hsluoyz) [inline docs/styles.module.css:9]: Where is this used?

File: static/img/embedding.png
Comment (hsluoyz) [inline static/img/embedding.png:1]: English

File: docs/overview.mdx [hunk 2]
```diff
@@ -1,18 +1,123 @@
 ---
 title: Overview
 description: Casibase Overview
-keywords: [casibase]
-authors: [casbin]
+keywords: [casibase,  ai,  knowledge database]
```
Comment (hsluoyz) [inline docs/overview.mdx:4]: one space

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: docs/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  margin: 0 auto;
+  transform: scale(0.85);
+  width: 600px;
+  box-shadow: 0 0 50px grey;
+  border-radius: 20px;
+}
```
Comment (Selflocking) [inline docs/styles.module.css:15]:
```suggestion
```

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 4]
```diff
@@ -19,8 +19,18 @@ module.exports = {
         "overview",
         {
           type: "link",
-          label: "PDF Download",
-          href: "/pdf",
```
Comment (Selflocking) [inline sidebars.js:23]: this should not be removed

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 4 above)
Comment (hsluoyz) [reply sidebars.js:23]: I think the PDF can be removed here, because it's not very useful in most cases..

**Review by Selflocking — APPROVED** [review]

**Comment by hsluoyz** [comment]
@kv-chiu why close?

**Pull request closed without merging**